### PR TITLE
Update data-driven-forms.json to wait

### PR DESCRIPTION
**Pull request by rvsia**
# Pull request motivation(s)

- DDF documentation page is based on next.js, most of pages are server-side rendered, however there are examples using query parameters to change the content of the page. The query parameter is not available on the first render in next.js so we have to wait until the page is updated.

### What is the current behaviour?

Algolia loads wrong data

### What is the expected behaviour?

Algolia should load updated data

cc @Hyperkid123

### CODE REVIEW

File: configs/data-driven-forms.json [hunk 1]
```diff
@@ -17,5 +17,7 @@
   "conversation_id": [
     "952910917"
   ],
-  "nb_hits": 3573
+  "nb_hits": 3573,
+  "js_render": true,
+  "js_wait": 2
```
Comment (s-pace) [inline configs/data-driven-forms.json:22]: Would it be possible to reduce as much as possible the time to wait?

Comment (rvsia) [reply configs/data-driven-forms.json:22]: I would like to, is there any way how to determine it? Or should I try some smaller value?

Comment (Hyperkid123) [reply configs/data-driven-forms.json:22]: The query params should be available almost instantly. The issue is that the router needs to just access the window property and set query values. It should take less than a second IMO.

Comment (s-pace) [reply configs/data-driven-forms.json:22]:
```suggestion
  "js_wait": 1
```
Let's give it a try with 1.

**Comment by s-pace** [comment]
Crawl done. Can you please check that the result is as expected?

**Comment by Hyperkid123** [comment]
@s-pace seems to be working. We are getting the search results.

**Pull request merged**